src/codegen/object.rs

```diff
@@ -249,6 +262,19 @@ fn generate_trait(w: &mut Write, env: &Env, analysis: &analysis::object::Info) -
     }
     try!(writeln!(w, "}}"));
 
+    if generate_display_trait {
+        try!(writeln!(
+            w,
+            "impl fmt::Display for {} {{",
+            analysis.name,
+        ));
+        // Generate Display trait implementation.
+        try!(writeln!(w, "\tfn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {{\n\
+                            \t\twrite!(f, \"{}\")\n\
+                          \t}}\n\
+            }}\n", analysis.name));
```

Comment:
For objects no empty line before and 2 empty lines at end

Reply:
"gtk/auto/check_button.rs" has `use std::fmt` but don't have `impl fmt::Display` as it don't have `CheckButtonExt`